package.json

```diff
@@ -1,5 +1,10 @@
 {
+  "scripts": {
+    "lint": "markdownlint '**/*.md'",
```

Comment:
I'm not sure if this is too hard to do, but it would be nice to change CI to use these scripts, so that we get testing for these lines?

Reply:
Agreed!

Reply:
Also the fact that the lint tool is installed on each run (minus any caching) means we could potentially run into a version mismatch and therefore a mismatch in output

Reply:
Gotta run to a meeting...started what I think is the right steps here: https://github.com/cdr/docs/pull/168/commits/3adf46fb1eca2e7f7acbc25b632f189eb5aaae1d